examples/decoupling_io/run.py

```diff
@@ -0,0 +1,85 @@
+# Since this is not in plugins we need to import it *before* doing anything else
+import importlib
+
+import click
+
+importlib.import_module("adapters")
+
+from components import feature_data, model_evaluation, model_training
+
+from hamilton import base, driver
+from hamilton.function_modifiers import source
+from hamilton.io.materialization import from_, to
+
+
+@click.group()
+def cli():
+    pass
+
+
+modules = {
+    "features_processed": [feature_data],
+    "model_trained": [feature_data, model_training],
+    "metrics_captured": [feature_data, model_training, model_evaluation],
+}
+
+
+def get_materializers(which_stage: str):
+    materializers = [
```

Comment:
Probably worth a comment to say this set represents `metrics_captured`,  as the logic in this function works, but could be simpler.